core/services/attributes_generator.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+require_relative 'configuration_generator'
+require_relative 'product_attributes'
+require_relative '../out'
+
+# The class provides methods for generating attributes for configuration generator
+module AttributesGenerator
```

Comment:
There is another module - ConfigurationGenerator. Why did you create another module instead of expanding the existing one?